### ISSUE TITLE
no surround output

### ISSUE THREAD

**Issue by enigma2k** [xbmc/audiodecoder.sacd#13]
Hello,

I'm using the addon on a Nvidia Shield Pro 2020 with Kodi Matrix and I am not getting any sound on the surround speakers in Kodi. All the files play, but just in stereo.

**Comment by Reemtsman** [comment]
Have you checked if your speaker setup is set to 5.1 under Kodi/Settings/Audio?
The default value here is 2.0 wich leads to only stereo output

**Comment by enigma2k** [comment]
yes, I have set it to 5.1 already. Kodi is working with movies in 5.1 without a problem. The AVR, however, is showing that it only gets stereo.

**Comment by fritsch** [comment]
Provide your debuglog. And What means: "Movies in 5.1 work without issues, but AVR shows stereo only" <- this sentence makes no sense.

**Comment by enigma2k** [comment]
I'm sorry you are right, the sentence did not make sense. What I mean is that the movies work without an issue in 5.1. It is only when I'm trying to play SACDs that they are in stereo. Kodi is showing it as 5.1 but the AVR only shows stereo.

**Comment by enigma2k** [comment]
I have uploaded a log file here: http://paste.kodi.tv/savigeqayo

**Comment by fritsch** [comment]
Log looks fine:

Loading additional tag info for file nfs://192.168.178.28/volume1/video/SACD/FLAC/surround51.flac <- you play this as a flac, right?

```
2021-03-06 12:44:56.964 T:27845   DEBUG <general>: Created Audiotrackbuffer with playing time of 160.416667 ms min buffer size: 92400 bytes
2021-03-06 12:44:56.984 T:27845    INFO <general>: CAESinkAUDIOTRACK::Initializing with: m_sampleRate: 48000 format: AE_FMT_S16LE (AE) method: PCM stream-type: PCM-STREAM min_buffer_size: 92400 m_frames: 1925 m_frameSize: 12 channels: 6
2021-03-06 12:44:56.984 T:27845   DEBUG <general>: CActiveAESink::OpenSink - AUDIOTRACK Initialized:
2021-03-06 12:44:56.984 T:27845   DEBUG <general>:   Output Device : AudioTrack (IEC)
2021-03-06 12:44:56.984 T:27845   DEBUG <general>:   Sample Rate   : 48000
2021-03-06 12:44:56.984 T:27845   DEBUG <general>:   Sample Format : AE_FMT_S16NE
2021-03-06 12:44:56.984 T:27845   DEBUG <general>:   Channel Count : 6
2021-03-06 12:44:56.984 T:27845   DEBUG <general>:   Channel Layout: FL, FR, FC, LFE, BL, BR
2021-03-06 12:44:56.984 T:27845   DEBUG <general>:   Frames        : 1925
2021-03-06 12:44:56.984 T:27845   DEBUG <general>:   Frame Size    : 12
```

**Comment by fritsch** [comment]
Can you try: https://www.dropbox.com/s/k53angu8enxi20u/cut.flac?dl=0 what works here?

**Comment by Reemtsman** [comment]
@enigma2k: Strange. Have you checked what happens when you switch from "All" to "Multicabnnel" in the Settings of the Add-on?
And one more thing: Are you sure, the SACD Iso in question really has a Multichannel Track? There are tons of SACD, just with a high resolution Stereo Track but no Multichannel..... just to be sure...

**Comment by enigma2k** [comment]
Thank you guys, I've managed to fix it! Really appreciate your time! It was actually some problem with the AVR setting even though I don't know exactly what caused it as I did not make any changes just switched through the sound modes and now it works no matter which mode is activated. I guess it must have been stuck before at some wrong sound mode that is not even normally accessible. (got the AVR used so maybe the previous owner made something)
